index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header-nav">
+        <h3>SheMer</h3>
+        <nav>
+            <ul>
+                <li><a href="/">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!--- Banner Section-->
+    <section class="bgimage" id="banner">
+
+        <div class="welcome-msg">
+            <h2 class="name">Hey We are SherMer</h2>
+            <p class="role">a web developer</p>
+        </div>
+
+    </section>
+    <!----Project Cards-->
+    <main class="container">
+        <h2>These are some of my projects</h2>
+
+        <div class="row">
+            <div class="col-sm-6">
+                <div class="card">
+                    <img class="card-img-top" alt="cards" width="400"
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"="Card image cap">
+                    <div class="card-block">
+                        <p class="card-text">Tribute Page</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-sm-6">
+                <div class="card">
+                    <img class="card-img-top" alt="cards" width="400"
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"="Card image cap">
+                    <div class="card-block">
+                        <p class="card-text">Tribute Page</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-sm-6">
+                <div class="card">
+                    <img class="card-img-top" alt="cards" width="400"
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"="Card image cap">
+                    <div class="card-block">
+                        <p class="card-text">Tribute Page</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </main>
+
```

Comment:
The structure looks wow. However, I think there are unused lines in the projects that we can do away with; for it to look more prettier. Right?😎 To do so, install an extension called prettier or you can run the linters code to clean it for you.